### DISCUSSION TITLE
使用tauri打包为本地应用

### DISCUSSION THREAD

**Discussion by cnkjw** [jaywcjlove/reference#646] [Ideas]
如题，是否可以考虑使用tauri打包为本地应用？我发现使用[Pake](https://github.com/tw93/Pake)这个项目应该可以轻松打包。不过我研究了半天还是没搞懂怎么用github action。看看有没有大神能出手搞定。

**Comment by jaywcjlove** [comment]
@K-JW https://github.com/tw93/Pake/releases/tag/V2.3.6 这里你可以下载，它已经支持了

<img width="467" alt="image" src="https://github.com/jaywcjlove/reference/assets/1680273/40ef8b0c-2676-4fcf-95dc-6b3e388a90be">

**Reply by cnkjw** [reply]
哈哈，好的。之前没注意到。非常感谢！
